tests/monit/test_monit_status.py

```diff
@@ -5,18 +5,45 @@
 
 import pytest
 
+from tests.common.utilities import wait_until
 from tests.common.helpers.assertions import pytest_assert
 
 logger = logging.getLogger(__name__)
 
 pytestmark = [
-    pytest.mark.topology('any')
+    pytest.mark.topology('any'),
+    pytest.mark.disable_loganalyzer
 ]
+@pytest.fixture
+def disable_lldp(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
```

Comment:
We can use `autouse` for the fixture. One example is: https://github.com/Azure/sonic-mgmt/blob/be84f77180d61c5e29bd93f53ed89102e4361f52/tests/ixia/conftest.py#L6

Reply:
@yozhao101 idea is to do this only for 2nd test case, If we do auto use both test case will have the fixture applied.
so i used explicit call.

Reply:
Before we stop the `lldpmgrd` process, do we need decide whether the lldp container is running or not？

Reply:
it should be running otherwise sanity will fail